README.md

```diff
@@ -49,26 +49,26 @@ Prepared images for local development in [LAMP devstack](https://en.wikipedia.or
     [`sodium`](https://www.php.net/manual/en/book.sodium.php),
     [`xsl`](https://www.php.net/manual/en/book.xsl.php) and
     [`zip`](https://www.php.net/manual/en/book.zip.php)
-- Apache activated modules: [`expires`](https://httpd.apache.org/docs/current/mod/mod_expires.html),
+- Apache modules: [`expires`](https://httpd.apache.org/docs/current/mod/mod_expires.html),
     [`headers`](https://httpd.apache.org/docs/current/mod/mod_headers.html) and
     [`rewrite`](https://httpd.apache.org/docs/current/mod/mod_rewrite.html)
 - Apache `DocumentRoot` changed to: `/var/www/html/www` (configurable by ENV)
-- PHP image has also pre-installed [Composer 2.2+](https://getcomposer.org/) and [Git 2.30+](https://git-scm.com/) to 
+- PHP image comes with [Composer 2.2+](https://getcomposer.org/) and [Git 2.30+](https://git-scm.com/) to 
     use it in guest shell  
-- MySQL properly configured to `utf8mb4` as default charset and optional support of Windows Host
-- optimized for correct support timezones
-- optimized for small image size a fast load
+- MySQL properly configured to `utf8mb4` as a default charset, optional support of Windows Host also available
```

Comment:
an optional

Reply:
- MySQL properly configured to *use* `utf8mb4` as a default charset, optional support of Windows Host *is* also available